test/integration/conversion/conversion_test.go

```diff
@@ -267,7 +119,7 @@ func TestConversion(t *testing.T) {
 			convertedObjMeta.GetObjectMeta().SetUID("")
 			convertedObjMeta.GetObjectMeta().SetSelfLink("")
 			convertedObjMeta.GetObjectMeta().SetResourceVersion("")
-			convertedObjMeta.GetObjectMeta().SetManagedFields([]metav1.ManagedFieldsEntry{})
+			convertedObjMeta.GetObjectMeta().SetManagedFields([]metav1.ManagedFieldsEntry(nil))
```

Comment:
I don't know why, but without this change the tests failed with a diff.

Reply:
Do they still? 🤔 weird...

Reply:
Perhaps I was mistaken. I've reverted this change